src/ui.tsx

```diff
@@ -558,6 +558,32 @@ function VisualizationOptions({ state }: { state: State }) {
   </>;
 }
 
+function OriginOptions({ state}: { state: State }) {
+  const dispatch = useContext(DispatchContext);
+  const device = Device(state.planOptions.hardware);
+  return <div className="flex">
+    <label title="Starting and final position of pen when plotting (x)">origin x:
```

Comment:
I hope it's not on inches

```suggestion
    <label title="Starting and final position of pen when plotting (x)">origin x (mm):
```

Reply:
I'm not sure it's mm so I left it dimensionless.  Might be pixels?

Reply:
All other dimensions are in mm